src/seedsigner/views/seed_views.py

```diff
@@ -375,6 +375,11 @@ def run(self):
         elif len(self.seed.passphrase) > 0:
             return Destination(SeedReviewPassphraseView)
         
+        elif self.seed.is_bip85_child_seed:
+            return Destination(SeedBIP85FinalizeView, view_args={
+                "bip85_data": self.seed.bip85_data,
```

Comment:
Why not just have `SeedBIP85FinalizeView` accept a `seed_num` like other Views and let it retrieve the `seed.bip85_data` on its own? A light `seed.is_bip85_child_seed` sanity check in there would catch any later misuses.

It's not a big deal but we should avoid passing around data that's already easily accessible elsewhere. Also just feels a little clunky and in some cases fragile to pass data to Views this way (though I'm sure there are places in the code where this does happen).